docs/paper/dev/api/entity-api/display-entities.mdx

```diff
@@ -0,0 +1,233 @@
+---
+slug: /dev/display-entities
+description: The display entity API and how to use it.
+---
+
+# Display Entities
+
+Added in 1.19.4, [display entities](https://minecraft.wiki/w/Display) are a way to display various
+things in the world, like blocks, items and text.
+
+By default, these entities have no hitbox, don't move, make sounds or take damage,
+making them the perfect for all kinds of applications, like holograms.
+
+## Types
+
+:::tip[Decorative displays]
```

Comment:
I feel like this should be under a "useful use cases" section at the bottom, after the basics are explained